cli/doall_cmd_test.go

```diff
@@ -45,7 +45,7 @@ func TestDoAll_cached_sample(t *testing.T) {
 	if o, err := gitCmd.CombinedOutput(); err != nil {
 		t.Fatal(string(o), err)
 	}
-	srcCmd = exec.Command("src", "do-all")
+	srcCmd = exec.Command("srclib", "do-all")
```

Comment:
I think this was an omission from a previous changeset, but I think this should be `srclib.CommandName`.